### PR TITLE
Added parser code for /proc/slabinfo

### PR DESCRIPTION
**Pull request by vishwanathjadhav**
Add a parser to fetch slab allocator details

Signed-off-by: vishawanathjadhav <vjadhav@redhat.com>

### CODE REVIEW

**Comment by bfahr** [comment]
@vishwanathjadhav I assume that this PR replaces #2425?  If so can you close the previous PR?

**Comment by vishwanathjadhav** [comment]
@bfahr - please review the PR

File: insights/parsers/slabinfo.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
+"""
```
Comment (bfahr) [inline insights/parsers/slabinfo.py]: @vishwanathjadhav please add a file in `docs/shared_parsers_catalog` to include this parser in the documentation.  See the [documentation guidelines](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html) for more information.

Comment (vishwanathjadhav) [reply insights/parsers/slabinfo.py]: Done

**Comment by vishwanathjadhav** [comment]
branch got corrupted due to forced push, https://github.com/RedHatInsights/insights-core/pull/2438

**Pull request closed without merging**